core/gnupg/gnupg.install

```diff
@@ -4,7 +4,7 @@ post_install() {
 }
 
 post_upgrade() {
-  if [[ $(vercmp $2 2.1.13-1) = -1 ]]; then
+  if [ $(vercmp $2 2.1.13-1) = -1 ]; then
```

Comment:
Here, I had an error that `[[` did not exist. So I made the change. I also checked and on my system, `/bin/[` exists and `/bin/[[` does not exist.